h2o-core/src/main/java/water/api/RapidsHandler.java

```diff
@@ -59,18 +64,39 @@ public RapidsSchemaV3 exec(int version, RapidsSchemaV3 rapids) {
     }
   }
 
+  private static Class[] getRapidsClasses(){
+    try {
+      InputStream is = Thread.currentThread().getContextClassLoader().getResourceAsStream("rapids.list");
```

Comment:
Hmm, expects that clasloaders are correctly setup - it does not be true that current thread ctx cl can see "rapids.list", in theory it would be better to use classloader of `this.getClass` (since we do not have any top-leve strategy for enforcing classloaders as for example Spark has).

Furthermore, just following comment from @michalkurka the functionality duplicates SPI - nevertheless, even for SPI, the CL issue above remains.